code/modules/surgery/organs/organ_internal.dm

```diff
@@ -290,6 +304,8 @@
 	say_mod = "buzzes"
 
 /obj/item/organ/internal/tongue/fly/TongueSpeech(var/message)
+	spawn(0)
+		speakDevilName(message)
```

Comment:
A proc that runs everytime you speak, just to make sure you didn't utter a truename?